### ISSUE TITLE
Mana particles don't spawn properly with Entity Culling installed

### ISSUE THREAD

**Issue by CammiePone** [williewillus/botania-fabric-issues#98]
Video for your viewing convenience, but for some reason none of the mana particles are spawning correctly. Specifically, they don't really go far at all, and in the case of mana spreaders, don't show up at all.

I don't know if it's a mod conflict or what, but it's really weird.

https://youtu.be/Tv3qbOsNeBw

**Comment by williewillus** [comment]
can you try with just botania?

**Comment by CammiePone** [comment]
I know with just Botania it works fine, but idk what's causing them to only partially render.

**Comment by williewillus** [comment]
can you try some more testing? Flat world with nothing around, etc. It's hard to know otherwise.

**Comment by williewillus** [comment]
Perhaps you have a mod that deflects projectile entities (which includes the burst)

**Comment by CammiePone** [comment]
No, the bursts are definitely hitting their targets, it's just the particles not making it the full way. I won't be able to do more testing for a few days, sadly, but I'll do it as soon as I can

**Comment by CammiePone** [comment]
![image](https://user-images.githubusercontent.com/16853086/142511677-c35f6c47-d209-4cd8-8cd2-82155c4f6e6d.png)

Sorry for the long wait, I even made a world for doing so and then ended up distracted with homework, and forgot to test again. Anyways, even in the superflat world (with the latest version of Botania that I just built from source) it still stops rendering particles. I'm really not sure what mod is responsible for this.

https://www.youtube.com/watch?v=EE3Gmf1o0Q4

**Comment by CammiePone** [comment]
So forget all of that, I found the mod responsible. Entity Culling

**Comment by Hubry** [comment]
Not only it causes issues on Forge (shoutouts to Twilight Forest getting multipart entities broken), but it causes them on Fabric too? Lmao

**Comment by CammiePone** [comment]
Seems that way, guess I'll be yeeting it. Want the issue closed or nah?

**Comment by Hubry** [comment]
Leave it for now, we might want to get this fixed.

(Also after a quick look, this might be a different entity culling mod than the one I mentioned above lol)

**Comment by CammiePone** [comment]
It would be this one:
https://www.curseforge.com/minecraft/mc-mods/entityculling
